_docs-sources/iac/usage/customizing-modules.md

```diff
@@ -0,0 +1,59 @@
+# Customizing Modules
+
+We strive to keep the Infrastructure as Code Library flexible and unopinionated, however there may be instances where you need to customize a module to your specific use case or a company policy that does not allow you to pull code from an external source.
```

Comment:
```suggestion
We strive to keep the Infrastructure as Code Library flexible and unopinionated, however there may be instances where you need to customize a module to your specific use case, or ensure compliance with a company policy that does not allow you to pull code from an external source.
```

Reply:
Added in [1bb5e8c](https://github.com/gruntwork-io/docs/pull/845/commits/1bb5e8c1cde7347864f815c74d604697d25dff36)